int32 robot_id
int32 client_id
string shelf
int32 items
